src/backend/joanie/core/serializers/client.py

```diff
@@ -763,12 +763,20 @@ class OrderGroupSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = models.OrderGroup
-        fields = ["id", "is_active", "nb_seats", "nb_available_seats"]
+        fields = [
+            "id",
+            "is_active",
+            "nb_seats",
+            "nb_available_seats",
+            "is_enabled",
+            "start",
+            "end",
+        ]
```

Comment:
IMO, we should no more return the `is_active` property in this client serializer. Furthermore, we could update the viewset queryset to exclude all order group with `is_active` property sets to `False`.